wiki/faq.md

```diff
@@ -473,13 +473,25 @@ But that is a little longer
 **Do not delete the data folder.** This contains important data like your keys which let you access your coins.
 
 ---
-# Backup
+# Backup & Migration
 
 ---
 ### What can I do if I didn't save a prior version of wallet.dat?
 
 The wallet has an automated backup in place that will make backups every 24 hours into the walletbackups folder inside of your data folder.
 
+---
+### How can I move my Gridcoin wallet to another machine?
+
+Note: These steps should work for any OS and for cross-OS migration. You should never run the same wallet on two machines at the same time because it looks to the network like an attack and your node may be banned. Additionally, you may lose coins and addresses as the wallets diverge over time. Don't do it.
+
+1. If you crunch BOINC to earn GRC, install BOINC on your new machine and attach to all your projects first. You must attach to all the same projects as before, you can set a project as "no new tasks" if you don't want to crunch it.
+2. Fully shut down the Gridcoin wallet, make sure it is not running in the system tray.
+3. Install the Gridcoin wallet on the new machine, fully shut it down after installation. You do not need to fully sync this wallet yet.
+4. Rename the wallet directory on the new machine to GridcoinResearch_old and copy the contents of your wallet directory from the original machine to the new machine in the same place

```

Comment:
You might want to remind folks where the data directory is.

Reply:
Fixed